tasks/install.yml

```diff
@@ -2,7 +2,7 @@
 
 - name: Install required packages
   become: yes
-  apt:
+  package:
```

Comment:
Very true, in that case can you remove the 1.9.4 refs as part of this PR? There is a reference in travis.yml and the min version in meta/main.yml.